lbry/tests/integration/test_claim_commands.py

```diff
@@ -470,17 +470,17 @@ class ChannelCommands(CommandTestCase):
         account2_id, account2 = new_account['id'], self.daemon.get_account_or_error(new_account['id'])
 
         # before sending
-        self.assertEqual(len(await self.daemon.jsonrpc_channel_list()), 3)
```

Comment:
This should still be valid, you can call channel_list() and get 3, right? Why did this one get removed?